BootloaderCorePkg/Stage1A/Ia32/Vtf0/Ia16/Real16ToFlat32.asm

```diff
@@ -111,5 +111,7 @@ jumpTo32BitAndLandHere:
     mov     fs, ax
     mov     gs, ax
     mov     ss, ax
+    /* Restore the BIST value to EAX register */
```

Comment:
"/* */" C-style comment cannot be used in ASM file.

Reply:
My bad! I was working recently with some .inc files. Got confused!

Reply:
My bad! I was working recently with some .inc files. Got confused!

Reply:
:)  No problem.   Sometimes it happens.